webpack/CVEsHostDetailsTab/CVEsHostDetailsTab.js

```diff
@@ -1,17 +1,37 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { translate as __ } from 'foremanReact/common/I18n';
+import { ScalprumComponent, ScalprumProvider } from '@scalprum/react-core';
+import { providerOptions } from '../common/ScalprumModule/ScalprumContext';
 
-const CVEsHostDetailsTab = ({ hostName }) => (
-  <div>
-    <h1>
-      {__('CVEs tab for host:')} {hostName}
-    </h1>
-  </div>
-);
+const CVEsHostDetailsTab = ({ systemId }) => {
+  const scope = 'vulnerability';
+  const module = './SystemDetailTable';
+  return (
+    <div className="rh-cloud-insights-vulnerability-host-details-component">
+      <ScalprumComponent scope={scope} module={module} systemId={systemId} />
+    </div>
+  );
+};
 
 CVEsHostDetailsTab.propTypes = {
-  hostName: PropTypes.string.isRequired,
+  systemId: PropTypes.string.isRequired,
+};
+
+const CVEsHostDetailsTabWrapper = ({ response }) => (
```

Comment:
The tab content is missing a bounding box. Its contents are rendered very close to the edges.

<img width="1124" height="504" alt="image" src="https://github.com/user-attachments/assets/9c065cc0-ceaa-4420-b23c-d90cd449543d" />